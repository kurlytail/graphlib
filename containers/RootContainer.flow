<?xml version="1.0" encoding="UTF-8"?>
<xmi:XMI xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:flow="http://www.brainspeedtech.com/swarm/flow" xmlns:notation="http://www.eclipse.org/gmf/runtime/1.0.2/notation">
  <flow:Flow xmi:id="_9Z2d8EW3EemKT5rL-yWemg" name="RootContainer">
    <cell xmi:type="flow:Cell" xmi:id="_Gbrq0EW4EemKT5rL-yWemg" input="_9BbcEEqHEemuQsfqsOazYw _pc6LMEqVEemuQsfqsOazYw _vPJPkEqVEemuQsfqsOazYw" name="RootContainer"/>
    <cell xmi:type="flow:Cell" xmi:id="_8K41oEqHEemuQsfqsOazYw" output="_9BbcEEqHEemuQsfqsOazYw" name="ButtonComponent"/>
    <cell xmi:type="flow:Cell" xmi:id="_mjgVIEqVEemuQsfqsOazYw" output="_pc6LMEqVEemuQsfqsOazYw" name="ButtonComponent"/>
    <cell xmi:type="flow:Cell" xmi:id="_uNSJ8EqVEemuQsfqsOazYw" output="_vPJPkEqVEemuQsfqsOazYw" name="ButtonComponent"/>
    <tube xmi:type="flow:Tube" xmi:id="_9BbcEEqHEemuQsfqsOazYw" from="_8K41oEqHEemuQsfqsOazYw" name="createCell" to="_Gbrq0EW4EemKT5rL-yWemg"/>
    <tube xmi:type="flow:Tube" xmi:id="_pc6LMEqVEemuQsfqsOazYw" from="_mjgVIEqVEemuQsfqsOazYw" name="createTube" to="_Gbrq0EW4EemKT5rL-yWemg"/>
    <tube xmi:type="flow:Tube" xmi:id="_vPJPkEqVEemuQsfqsOazYw" from="_uNSJ8EqVEemuQsfqsOazYw" name="reverse" to="_Gbrq0EW4EemKT5rL-yWemg"/>
  </flow:Flow>
  <notation:Diagram xmi:id="_9Z7WcEW3EemKT5rL-yWemg" type="Flow" element="_9Z2d8EW3EemKT5rL-yWemg" name="RootContainer.flow" measurementUnit="Pixel">
    <children xmi:type="notation:Shape" xmi:id="_Gby_kEW4EemKT5rL-yWemg" type="2001" element="_Gbrq0EW4EemKT5rL-yWemg" fontName=".SF NS Text">
      <children xmi:type="notation:DecorationNode" xmi:id="_GbzmoEW4EemKT5rL-yWemg" type="5001"/>
      <layoutConstraint xmi:type="notation:Bounds" xmi:id="_Gby_kUW4EemKT5rL-yWemg" x="387" y="234"/>
    </children>
    <children xmi:type="notation:Shape" xmi:id="_8K6DwEqHEemuQsfqsOazYw" type="2001" element="_8K41oEqHEemuQsfqsOazYw" fontName=".SF NS Text">
      <children xmi:type="notation:DecorationNode" xmi:id="_8K6DwkqHEemuQsfqsOazYw" type="5001"/>
      <layoutConstraint xmi:type="notation:Bounds" xmi:id="_8K6DwUqHEemuQsfqsOazYw" x="208" y="104"/>
    </children>
    <children xmi:type="notation:Shape" xmi:id="_mjiKUEqVEemuQsfqsOazYw" type="2001" element="_mjgVIEqVEemuQsfqsOazYw" fontName=".SF NS Text">
      <children xmi:type="notation:DecorationNode" xmi:id="_mjixYEqVEemuQsfqsOazYw" type="5001"/>
      <layoutConstraint xmi:type="notation:Bounds" xmi:id="_mjiKUUqVEemuQsfqsOazYw" x="207" y="432"/>
    </children>
    <children xmi:type="notation:Shape" xmi:id="_uNT_IEqVEemuQsfqsOazYw" type="2001" element="_uNSJ8EqVEemuQsfqsOazYw" fontName=".SF NS Text">
      <children xmi:type="notation:DecorationNode" xmi:id="_uNT_IkqVEemuQsfqsOazYw" type="5001"/>
      <layoutConstraint xmi:type="notation:Bounds" xmi:id="_uNT_IUqVEemuQsfqsOazYw" x="496" y="425"/>
    </children>
    <styles xmi:type="notation:DiagramStyle" xmi:id="_9Z7WcUW3EemKT5rL-yWemg"/>
    <edges xmi:type="notation:Connector" xmi:id="_9BcDIEqHEemuQsfqsOazYw" type="4001" element="_9BbcEEqHEemuQsfqsOazYw" source="_8K6DwEqHEemuQsfqsOazYw" target="_Gby_kEW4EemKT5rL-yWemg" routing="Rectilinear">
      <children xmi:type="notation:DecorationNode" xmi:id="_9BcqMEqHEemuQsfqsOazYw" type="6001">
        <layoutConstraint xmi:type="notation:Location" xmi:id="_9BcqMUqHEemuQsfqsOazYw" x="-40" y="8"/>
      </children>
      <styles xmi:type="notation:FontStyle" xmi:id="_9BcDIUqHEemuQsfqsOazYw" fontName=".SF NS Text"/>
      <bendpoints xmi:type="notation:RelativeBendpoints" xmi:id="_9BcDIkqHEemuQsfqsOazYw" points="[19, 19, -122, -119]$[138, 138, -3, 0]"/>
      <targetAnchor xmi:type="notation:IdentityAnchor" xmi:id="_9Bd4UEqHEemuQsfqsOazYw" id="(0.046875,0.675)"/>
    </edges>
    <edges xmi:type="notation:Connector" xmi:id="_pc8AYEqVEemuQsfqsOazYw" type="4001" element="_pc6LMEqVEemuQsfqsOazYw" source="_mjiKUEqVEemuQsfqsOazYw" target="_Gby_kEW4EemKT5rL-yWemg" routing="Rectilinear">
      <children xmi:type="notation:DecorationNode" xmi:id="_pc8nckqVEemuQsfqsOazYw" type="6001">
        <layoutConstraint xmi:type="notation:Location" xmi:id="_pc8nc0qVEemuQsfqsOazYw" y="40"/>
      </children>
      <styles xmi:type="notation:FontStyle" xmi:id="_pc8ncEqVEemuQsfqsOazYw" fontName=".SF NS Text"/>
      <bendpoints xmi:type="notation:RelativeBendpoints" xmi:id="_pc8ncUqVEemuQsfqsOazYw" points="[17, -19, -154, 178]$[164, -177, -7, 20]"/>
    </edges>
    <edges xmi:type="notation:Connector" xmi:id="_vPJPkUqVEemuQsfqsOazYw" type="4001" element="_vPJPkEqVEemuQsfqsOazYw" source="_uNT_IEqVEemuQsfqsOazYw" target="_Gby_kEW4EemKT5rL-yWemg" routing="Rectilinear">
      <children xmi:type="notation:DecorationNode" xmi:id="_vPJPlEqVEemuQsfqsOazYw" type="6001">
        <layoutConstraint xmi:type="notation:Location" xmi:id="_vPJPlUqVEemuQsfqsOazYw" y="40"/>
      </children>
      <styles xmi:type="notation:FontStyle" xmi:id="_vPJPkkqVEemuQsfqsOazYw" fontName=".SF NS Text"/>
      <bendpoints xmi:type="notation:RelativeBendpoints" xmi:id="_vPJPk0qVEemuQsfqsOazYw" points="[-12, -19, 96, 154]$[-107, -170, 1, 3]"/>
      <targetAnchor xmi:type="notation:IdentityAnchor" xmi:id="_vPKdsEqVEemuQsfqsOazYw" id="(0.65625,0.925)"/>
    </edges>
  </notation:Diagram>
</xmi:XMI>
