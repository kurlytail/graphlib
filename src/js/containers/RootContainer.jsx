// @flow

import React, { Component } from 'react';

import { BrowserRouter as Router } from 'react-router-dom';

import { Route } from 'react-router-dom';

class RootContainer extends Component {
    render() {
        const Container = <div />;

        return <Router>{Container}</Router>;
    }
}

export default RootContainer;
