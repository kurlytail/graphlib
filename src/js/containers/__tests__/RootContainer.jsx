import React from 'react';
import Renderer from 'react-test-renderer';

import RootContainer from '../RootContainer';

describe('# RootContainer', () => {
    describe('## constructor', () => {
        it('### should create default object', () => {
            const container = Renderer.create(<RootContainer />);
            expect(container.toJSON()).toMatchSnapshot();
        });
    });
});
