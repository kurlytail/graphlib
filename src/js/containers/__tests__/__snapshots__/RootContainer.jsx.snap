// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# RootContainer ## constructor ### should create default object 1`] = `<div />`;
